chapter11/drjt/ex1.pm

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/perl
+use 5.010;
+use strict;
+use warnings;
+
+package ex1;
+
+use constant { DAY_NAMES => [qw( ark dip wap sen pop sep kir )],
+               MONTH_NAMES => [qw( diz pod bod rod sip wax lin sen kun fiz nap dep )],
+               };
```

Comment:
Your references here are constant, but the data they reference isn't. Try to avoid using constants like this.